app-creator/client/src/widgets/templates-tab/templates-tab.ts

```diff
@@ -90,9 +105,22 @@ export class TemplatesTab extends connect(store)(LitElement) {
    */
   @property({ type: String }) deviceFilter = DeviceType.ALL;
 
+  /**
+   * Reference to template change confirmation dialog.
+   */
+  @query('#template-change-confirmation-dialog')
+  templateChangeConfirmationDialog!: PaperDialogElement;
+
+  stateChanged(state: AppCreatorStore) {
```

Comment:
Is this function used outside of the class? If not, let's mark it as private. Similar for the other functions in this class.

Reply:
The stateChanged method is provided to us by the store to listen to changes. Its interface requires that it is public. This is the error that I get when I set it to private. The other methods however can be changed to private :)
```
Class 'TemplatesTab' incorrectly extends base class '{ _storeUnsubscribe: Unsubscribe; ...}'.
Property 'stateChanged' is private in type 'TemplatesTab' but not in type '{ _storeUnsubscribe: Unsubscribe; 
connectedCallback(): void; disconnectedCallback(): void; stateChanged(_state: AppCreatorStore): void; readonly isConnected: boolean; }'.
```